fanctrl.py

```diff
@@ -81,6 +81,11 @@ def initParser(self):
                 choices=["ectool"],
                 default="ectool"
             )
+            runCommand.add_argument(
+                "--no-battery",
+                help="disable ectool checking the battery sensor",
```

Comment:
Maybe should we make the "no-battery" option more general and not ectool specific?